CodeStarterCamp_Week2/Method.swift

```diff
@@ -1,11 +1,18 @@
 
 import Foundation
 
-func printLottoResult(){
-    if correctNumbers.isEmpty {
-        print("아쉽지만 겹치는 번호가 없습니다.")
-    } else {
-        print("축하합니다! 겹치는 번호는 \(convertToString(correctNumbers)) 입니다! ")
+func findLotto(number: Int) {
+    let numberToString = convertToString([number]) + "회차"
+    
+    if let lottoNumbers = lottoDictionary[numberToString]{
+        print("\(numberToString)의 로또 당첨 번호는 \(convertToString(lottoNumbers)) 입니다.")
+    }
+}
```

Comment:
- 여기서도, 메서드의 네이밍을 손봤습니다.
  - 이 메서드의 역할은, 파라미터로 들어오는 숫자에 해당하는 로또 티켓을 print 해주는 것이죠?
  - 따라서, 그냥 그 티켓을 `보여준다`는 의미를 담아 `"show"` 동사를 사용했어요. 😄 
  - 그리고 여기 파라미터도 똑같은 `Int 타입`을 받지만, count 가 아니라 number 를 그대로 쓰는 이유는 아시겠죠?
  - 여기서는 `"개수"`의 의미가 아니라, `"번째"`의 의미라서 그렇습니다! [(참고) 기수 vs 서수](http://weteacher.net/math/number/number16.htm)

- 메서드의 파라미터를 추가했습니다.
  - 위 코멘트에서, 제가 `객체지향` 얘기 했었어요. 메서드들 끼리 협력하는 관점에서 생각해보라구요 ㅎㅎㅎ
  - 이 메서드의 역할은, `X번째 로또 티켓`을 출력해주는 것이고, 그러려면 일단 `로또 티켓들`을 어디선가 참조해야 겠죠?
  - 그냥 로또 티켓들을 전역 변수로 놔두고 해도 작동은 하겠지만, 전역 변수를 불필요하게 많이 만들 필요는 없으니까요. 😅 

- `예외 처리`를 추가했습니다.
  - 요구 사항에는 없지만, if문이 쓰이면 else문의 사용도 고민해보면 좋아요.
  - 구매한 로또 티켓의 개수(count) 보다 보여달라고 요청한 티켓의 number 가 크면, 문제가 생기겠죠!
  - 이런 식으로, 간단하게 예외처리 고민해보는 점도 좋겠어요. 😄 

- convertToString 재사용을 과하게 할 필요 없어요.
  - 재사용하는 건 좋지만, `문자열 보간법`으로 충분히 할 수 있는 정도인데, 메서드를 끌고 올 필요는 없죠. ㅎㅎㅎ
  - 코드로 비교해보세요.

```suggestion
func showLottoTicket(number: Int, in tickets: [String: [Int]]) {
    if let lottoTicket = tickets["\(number)회차"] {
        print("\(number)회차의 로또 당첨 번호는 \(convertToString(lottoTicket)) 입니다.")
    } else {
        // 예외 처리 해주는 센스
        print("해당 회차의 로또 당첨 번호를 찾을 수 없습니다.")
    }
}
```